Description: Open the modules to remove the reflective access warnings on the command line
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/jaxb-ri/bundles/ri/src/main/resources/bin/xjc.zip.sh
+++ b/jaxb-ri/bundles/ri/src/main/resources/bin/xjc.zip.sh
@@ -74,4 +74,4 @@
     JAVA=java
 fi
  
-$JAVA $XJC_OPTS -cp "$CLASSPATH" com.sun.tools.xjc.Driver "$@"
+$JAVA $XJC_OPTS --add-opens=java.base/java.lang=ALL-UNNAMED -cp "$CLASSPATH" com.sun.tools.xjc.Driver "$@"
